spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -819,6 +839,26 @@ public void testPartitionedImportFromEmptyPartitionDoesNotThrow() {
               RowFactory.create(4, "Will Doe", "facilities", toDate("2021-01-02"))),
           new StructType(dateStruct)).repartition(2);
 
+  private static final StructField[] timeStampstruct = {
+      new StructField("id", DataTypes.IntegerType, true, Metadata.empty()),
+      new StructField("name", DataTypes.StringType, true, Metadata.empty()),
+      new StructField("dept", DataTypes.StringType, true, Metadata.empty()),
+      new StructField("ts", DataTypes.TimestampType, true, Metadata.empty())
+  };
+
+  private static Timestamp toTimestamp(String value) {
+    return new Timestamp(DateTime.parse(value).getMillis());
```

Comment:
This is not a reliable way to convert to a timestamp because Java Date and SQL Timestamp have internal time zone logic. Instead, please use Literal or parse values directly.